app/src/main/res/values/dimens.xml

```diff
@@ -74,8 +74,8 @@
     <dimen name="event_details_divider">1.2dp</dimen>
     <dimen name="divider_margin_bottom">20dp</dimen>
     <dimen name="divider_margin_top">12dp</dimen>
-    <dimen name="collapsing_toolbar_height">220dp</dimen>
-    <dimen name="details_header_margin_top">40dp</dimen>
+    <dimen name="collapsing_toolbar_height">320dp</dimen>
+    <dimen name="details_header_margin_top">100dp</dimen>
```

Comment:
Add dimensions in dimension resources file.

Reply:
@liveHarshit this is dimensions inside dimens.xml file... Can you please clarify your review?